templates/latest/cri-o/cri-o.spec

```diff
@@ -80,6 +80,12 @@ install -p -m 644 %{archive_root}/contrib/11-crio-ipv4-bridge.conflist %{buildro
 
 # Fix the prefix in crio.service
 sed -i 's;/usr/local/bin;/usr/bin;g' %{archive_root}/contrib/crio.service
+
+# Fix the /etc/sysconfig path for debian based distributions
+%if "%{_vendor}" == "debbuild"
+sed -i 's;/etc/sysconfig/crio;/etc/default/crio;g' %{archive_root}/contrib/crio.service
```

Comment:
I think, the `/etc/default/crio` file will still be missing.

We could copy it from https://github.com/cri-o/cri-o/blob/main/contrib/sysconfig/crio, provided that we also update it with all the expected environment variables. Unless we just leave it to the user to create.

Reply:
So, do you expect users to have to go and read the systemd unit file if they want to use it?

Reply:
Somehow, yes. I'm not sure. we can add it later on but I don't see much value in it to be honest. What do other @cri-o/cri-o-maintainers think?

Reply:
Having a cursory look at my test system. Not all packages claim the file to manage it, but at least install a relevant file when the package is installed for users' convenience per:

```console
root@worker-node01:~# for f in /etc/default/* ; do dpkg -S $f ; done
amd64-microcode: /etc/default/amd64-microcode
apport: /etc/default/apport
dpkg-query: no path found matching pattern /etc/default/console-setup
dpkg-query: no path found matching pattern /etc/default/crio
cron: /etc/default/cron
cryptsetup: /etc/default/cryptdisks
dbus: /etc/default/dbus
dpkg-query: no path found matching pattern /etc/default/grub
grub-common: /etc/default/grub.d
dpkg-query: no path found matching pattern /etc/default/grub.ucf-dist
intel-microcode: /etc/default/intel-microcode
irqbalance: /etc/default/irqbalance
dpkg-query: no path found matching pattern /etc/default/keyboard
dpkg-query: no path found matching pattern /etc/default/kubelet
dpkg-query: no path found matching pattern /etc/default/locale
dpkg-query: no path found matching pattern /etc/default/mdadm
networkd-dispatcher: /etc/default/networkd-dispatcher
open-iscsi: /etc/default/open-iscsi
pollinate: /etc/default/pollinate
rsync: /etc/default/rsync
openssh-server: /etc/default/ssh
dpkg-query: no path found matching pattern /etc/default/sysstat
ufw: /etc/default/ufw
passwd: /etc/default/useradd
```

This is the expectation I would also have on a Debian-based system. We don't have to do it, of course, but it would be nice to have done it, since we offer support for the `/etc/default/crio`.

Reply:
Okay let's do it in https://github.com/cri-o/packaging/pull/119